react/src/lib/components/Menu/hooks/useContainerDimensions.ts

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { Size } from "../types/size";
+
+export const useContainerDimensions = (
```

Comment:
Replaced it with `useSize()`. However, made it globally available, so it can be used in other components now. If not in use, it won't be included in the build JS file anyways (so no concerns about size).